Link list updates v2 (#1673)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/link-list.mdx

<AnchorLinks>

<AnchorLink>Overview</AnchorLink>
<AnchorLink>Variations</AnchorLink>
<AnchorLink>Content guidance</AnchorLink>
<AnchorLink>Resources</AnchorLink>
<AnchorLink>Feedback</AnchorLink>

</AnchorLinks>

## Overview

> The Link list displays links in a group format, with subtle divider lines, an optinal heading and the icon shown to the right of the link text. It is

Reviewer: ```suggestion
The Link list displays links in a group format, with subtle divider lines, an optional heading and the icon shown to the right of the link text. It is
```

typically used for resource links on the side of the main page content.

<Row>
<Column colMd={8} colLg={8}>

![Image of link list component](../../images/component/link-list/link-list-overview.png)

</Column>
</Row>

## Variations

### End of section

> This variation displays is similar to the default variation and can support up to 16 CTAs. It is typically used at the end of a paragraph or content section to list a number of CTAs for

Reviewer: ```suggestion
This variation is similar to the default variation and can support up to 16 CTAs. It is typically used at the end of a paragraph or content section to list a number of CTAs for
```

the user and will spread the links out across the grid based on the number of CTAs.

![Image of link list end of section component](../../images/component/link-list/link-list-end-of-section.png)

### Horizontal

This variation displays links in a horizontal row with an icon. It is typically used within the main body of the page.

<Row>
<Column colMd={8} colLg={8}>

![Image of link list horizontal component](../../images/component/link-list/link-list-horizontal.png)

</Column>
</Row>

### Vertical

> This variation displays links as a minimal, vertical list with an icon. It is typically used for resource links on the

Reviewer: ```suggestion
This variation displays a set links in a vertical list, and each link is displayed with an icon. It is typically used for resource links on the
```
